pypeit/core/flux_calib.py

```diff
@@ -420,6 +420,30 @@ def get_standard_spectrum(star_type=None, star_mag=None, ra=None, dec=None):
             # vega is V=0.03
             std_dict['flux'] = vega_data['col2'] * 10**(0.4*(0.03-star_mag)) / PYPEIT_FLUX_SCALE * \
                                units.erg / units.s / units.cm ** 2 / units.AA
+        elif 'PHOENIX' in star_type:
+            msgs.info('Getting PHOENIX 10000 K, logg = 4.0 spectrum')
+            ## Vega model from TSPECTOOL
+            vega_file = data.Paths.standards / 'PHOENIX_10000K_4p0.dat'
```

Comment:
I think it's best to store this as a `.fits` file. It will be faster to load, and it's better to store these large files in binary format.